util/chplenv/chpl_arch.py

```diff
@@ -571,8 +587,9 @@ def _main():
 
     if options.compflag:
         stdout.write("{0}=".format(flag))
-    stdout.write("{0}\n".format(arch))
-
+        stdout.write("{0}\n".format(arch))
+    else:
+        stdout.write("{0}\n".format(arch))
```

Comment:
What's the motivation for this change?

Reply:
Mattered in earlier version, thought I undid that, will fix